corgi/tasks/monitoring.py

```diff
@@ -75,7 +75,8 @@ def upsert_interval_task(module, task, hours=None, minutes=None):
         upsert_cron_task("yum", "fetch_unprocessed_yum_relations", hour=9, minute=0)
         upsert_cron_task("manifest", "update_manifests", hour=10, minute=0)
         upsert_cron_task("manifest", "collect_static", hour=11, minute=0)
-        upsert_cron_task("monitoring", "email_failed_tasks", hour=11, minute=45)
+        upsert_cron_task("managed_services", "refresh_service_manifests", hour=11, minute=45)
+        upsert_cron_task("monitoring", "email_failed_tasks", hour=12, minute=45)
```

Comment:
Sorry I didn't catch this earlier - if this managed services are just streams in prod-defs, we should probably refresh_service_manifests before we update_manifests. Otherwise the manifest we generate and serve might have stale data:
```suggestion
        upsert_cron_task("managed_services", "refresh_service_manifests", hour=10, minute=0)
        upsert_cron_task("manifest", "update_manifests", hour=11, minute=0)
        upsert_cron_task("manifest", "collect_static", hour=12, minute=0)
        upsert_cron_task("monitoring", "email_failed_tasks", hour=12, minute=45)
```

Reply:
Ack, shuffled around!